error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/LexicographicalAnnotationListingTest.java

```diff
@@ -106,6 +119,13 @@ void identification() {
             "  @Bar",
             "  @Foo(ints = {1, 2})",
             "  A unsortedRepeatableAnnotation();",
+            "",
+            "  // BUG: Diagnostic matches: X",
```

Comment:
Forgot to comment the unrelated improvement, but we can just use `Diagnostic contains:` here instead of the `X`.